app.rb

```diff
@@ -19,80 +18,20 @@ def list_books
   end
 
   def list_people
-    people = [*@teacher, *@student]
-    people.each_with_index { |person, i| print "(#{i}) Name: \"#{person.name}\", Age: \"#{person.age}\"\n" }
-  end
-
-  def create_person
-    print 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
-    input = gets.chomp.to_i
-    case input
-    when 1
-      print 'Age: '
-      age = gets.chomp.to_i
-      print 'Name: '
-      name = gets.chomp
-
-      print 'Do you have parental permission? [Y/N]: '
-      parent_permission = gets.chomp
-      case parent_permission.capitalize
-      when 'Y'
-        true
-      else
-        false
-      end
-      print 'Enter classroom:'
-      classroom = gets.chomp
-      new_student = Student.new(age, classroom)
-      new_student.name = name
-      @student.push(new_student)
-      puts 'Student has been created successfully'
-    when 2
-      print 'Age: '
-      age = gets.chomp.to_i
-      print 'Name: '
-      name = gets.chomp
-      print 'Enter specialization:'
-      specialization = gets.chomp
-      new_teacher = Teacher.new(age, specialization)
-      new_teacher.name = name
-      @teacher.push(new_teacher)
-      puts 'Teacher has been created successfully'
-    else
-      puts 'Wrong input'
+    # @people = [*@teacher, *@student]
+    # people.each_with_index do |person, i|
```

Comment:
- **Please remove all commented code in this class. This is because you need to have better practice when writing code(Clean, DRY, kISS, YAGNI)**